app/assets/stylesheets/components/beta_tag/_beta_tag.scss

```diff
@@ -0,0 +1,11 @@
+.beta-tag {
+  @include inline-block();
+  @include body(18, 24);
+  font-weight: 700;
+  position: relative;
+  bottom: -$baseline-unit*5;
```

Comment:
This seems a bit of an odd way to position it. Shouldn't we just add a margin to the global layout or something?